package.json

```diff
@@ -34,33 +39,29 @@
     "validate": "kcd-scripts validate"
   },
   "dependencies": {
-    "@babel/core": "^7.12.16",
-    "@babel/preset-env": "^7.12.16",
-    "@babel/preset-react": "^7.12.13",
-    "@babel/preset-typescript": "^7.12.16",
-    "@babel/runtime": "^7.12.13",
+    "@babel/runtime": "^7.13.10",
+    "@esbuild-plugins/node-resolve": "0.0.14",
+    "@fal-works/esbuild-plugin-global-externals": "^2.1.1",
     "@mdx-js/mdx": "^1.6.22",
     "@mdx-js/react": "^1.6.22",
-    "@rollup/plugin-babel": "^5.2.3",
-    "@rollup/plugin-commonjs": "^17.1.0",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-node-resolve": "^11.1.1",
+    "esbuild": "0.9.0",
     "gray-matter": "^4.0.2",
-    "rollup": "^2.39.0",
-    "rollup-plugin-terser": "^7.0.2"
+    "remark-frontmatter": "^3.0.0",
+    "remark-mdx-frontmatter": "^1.0.0"
```

Comment:
this one can be removed.

remark-frontmatter also needs to be downgraded 1 version I think to work with mdx-js/mdx?